sonaranalyzer-dotnet/its/expected/Nancy/Nancy-{34576216-0DCA-4B0F-A0DC-9075E75A676F}-S5773.json

```diff
@@ -7,7 +7,7 @@
 "uri":  "sources\Nancy\src\Nancy\DefaultObjectSerializer.cs",
 "region":  {
 "startLine":  54,
-"startColumn":  28,
+"startColumn":  38,
```

Comment:
shouldn't a secondary location be raised?

Reply:
or is this related to
```
                // An issue can appear multiple times for the same node while exploring the exploded graph.
                // In this case the location is added one single time to avoid duplicates.
```

but in that case, two issues should have been replaced by one, right?

Reply:
In this case no since the secondary locations are added only for classes which derive from `SerializationBinder` or `JavaScriptTypeResolver`